docs/src/rsh/errors/index.md

```diff
@@ -2116,6 +2116,22 @@ Maybe(UInt).Some(5).match({
 This error indicates that you have inspected the details about a publication, such as via `{!rsh} didPublish()`, before there has been a publication.
 This is impossible, so the expression must be moved after the first publication.
 
+For example, the code below erroneously puts `{!rsh} didPublish()` before `{!rsh} publish`:
+
+```reach
+  Alice.didPublish(handAlice);
```

Comment:
This program is broken for a different reason... you can only call `didPublish` inside of an `only` block. The real way to get this error is to call it before `publish` but after `init`